src/components/Migration/MigrationModal.tsx

```diff
@@ -368,11 +407,25 @@ function MigrationModal({ open, handleClose }: { open: boolean; handleClose: any
                     <Typography>
                       {isMigrationComplete || !oldAssetsDetected
                         ? "Close"
-                        : txnButtonText(pendingTransactions, "migrate_all", t`Migrate`)}
+                        : txnButtonText(
+                            pendingTransactions,
+                            "migrate_all",
+                            t`Migrate all to ${isGOHM ? "gOHM" : "sOHM"}`,
```

Comment:
you can't wrap a code block in `t` or else it throws this funkiness, fyi.
![image](https://user-images.githubusercontent.com/80423742/149262723-c927607a-1296-4b9a-8260-73572f89cb97.png)

Reply:
fix: `${t`Migrate all to`} ${isGOHM ? "gOHM" : "sOHM"}`,